### ISSUE TITLE
getPallete running multiple times

### ISSUE THREAD

**Issue by eybarta** [akfish/node-vibrant#34]
Hi thanks for awesome plugin..

I'm running this code:
```

console.log("this happends once");

Vibrant.from(this.image).getPalette((err, palette) => {

        console.log("this happends 3 times")

        for (color in palette) {
               if (this.colors.length<5 && !!palette[color].rgb)
                    this.colors.push(palette[color].rgb.join(','))
        }
})

```
the Vibrant code works fine except it runs multiple times instead of once, I'm not sure
what I'm doing wrong here.. could you please help me figure out this issue?

thnx!

**Comment by akfish** [comment]
I cannot reproduce this issue on my end. The callback is called exactly once.
You could try adding a break point inside the callback or print the stack trace to identify the callsites and see where the extra calls originate.

**Comment by Uriziel01** [comment]
I'm having the exact same issue. Will try to prepare some jsFiddle example later today.

**Comment by akfish** [comment]
I just published v3.0.0-alpha.1. Could you see if this issue persists in the new version?

**Comment by Uriziel01** [comment]
Yes, now it works like a charm (triggering only once). 👍 

It should be a separate issue but I just want to drop a quick note that it bothers me that loading **v3.0.0.alpha.1** using async import:
``` js
System.import('node-vibrant').then(function (importedObject) {
    self.vibrant = importedObject;
});
```
will not work because object returned from `System.import` is an empty object, I have to use it like so:
``` js
System.import('node-vibrant').then(function (importedObject) {
    self.vibrant = Vibrant;
});
```

It was working correctly in previous vibrant versions.

**Comment by akfish** [comment]
Good to know.

I opened a separate issue to track the other issue.
